packages/itwinui-react/src/core/Carousel/CarouselSlide.tsx

```diff
@@ -52,6 +53,8 @@ export const CarouselSlide = React.forwardRef((props, ref) => {
 
   const refs = useMergedRefs(intersectionRef, ref);
 
+  useInertPolyfill();
```

Comment:
Since we use `inert` in multiple places, I wonder if we should do this in `ThemeProvider` instead?

Reply:
Yeah it's a tradeoff. I'm ok with doing it in `ThemeProvider` because:
- it's less code for us to maintain
- it's a small download and we're lazy importing it (so it doesn't block the initial render)
- it doesn't affect the majority of users (who are likely using an up-to-date browser)